### ISSUE TITLE
Attach to Chrome Canary Error

### ISSUE THREAD

**Issue by ajsb85** [Microsoft/vscode-chrome-debug#208]
**VS Code - Debugger for Chrome - v0.5.1**

![image](https://cloud.githubusercontent.com/assets/663460/17582184/82485764-5faa-11e6-827d-f6ac9feb110b.png)

```JSON
{
   "Browser": "Chrome/54.0.2802.0",
   "Protocol-Version": "1.1",
   "User-Agent": "Mozilla/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/54.0.2802.0 Safari/537.36",
   "WebKit-Version": "537.36 (@fc6aaca4ed6ff4f050e6f5c7fb19af85da8db574)"
}
```

**Comment by roblourens** [comment]
Tracking in Microsoft/vscode-chrome-debug-core#76

**Comment by roblourens** [comment]
Fixed in 0.5.2!

**Comment by roblourens** [comment]
I should mention that it won't be at total parity since it's using limited information coming from Chrome via a deprecated API - more work to do but should unblock people.
